__tests__/app.test.js

```diff
@@ -34,4 +37,15 @@ describe('GET /api/topics', () => {
             expect(body.msg).toBe('Not Found');
           });
       });
-});
\ No newline at end of file
+});
+
+describe('GET /api', () => {
+    test('GET: 200 - responds with a JSON object describing all endpoints', async () => {
```

Comment:
You're not using async/await so this `async` keyword isn't really necessary here :)